f5_openstack_agent/lbaasv2/drivers/bigip/resource_manager.py

```diff
@@ -588,8 +588,11 @@ def _update(self, bigip, vs, old_listener, listener, service):
         if vs.get('customized', None):
             self._create_http_profile(bigip, listener, vs)
 
-        # at least one attribute in vs' patch body as icontrol patch
-        vs['description'] = listener.get('name', '')
+        # If no vs property to update, do not call icontrol patch api.
+        # This happens, when vs payload only contains 'customized'.
+        if sorted(vs.keys()) == ['name', 'partition'] or \
+           sorted(vs.keys()) == ['customized', 'name', 'partition']:
```

Comment:
Thanks, I forgot there's still a 'customized' in vs. Maybe at line 590: `del vs['customized']` within the if can make the code cleaner.

Reply:
Done